### ISSUE TITLE
5th argument

### ISSUE THREAD

**Issue by shranet** [FatihBAKIR/UnityPacker#2]
I want to skip directories started at ".DS_Store". But, the app does not skipped directories started with .DS inside directory.

Example:
./.DS_Store
./Editor/Test.cs
./Plugins/.DS_Store
...

Build:
mono UnityPacker.exe . "Name" no "Assets/" "gitignore,md" ".git,.DS"

Exported:
./Editor/Test.cs
./Plugins/.DS_Store
...

**Comment by FatihBAKIR** [comment]
You are right, the directory omission stuff isn't recursive. I'll look into it.
